src/Compilers/Core/SharedCollections/ArrayBuilder.cs

```diff
@@ -75,6 +75,10 @@ public int Count
             {
                 return _builder.Count;
             }
+            set
```

Comment:
Is this used in the same way as Clip or it is to support expansion of the builder with uninitialized values as well?